projects/1.21-fabric/assets/newworld/newworld/lang/zh_cn.json

```
@@ -0,0 +1,86 @@
+{
+  "advancements.adventure.collect_ancient_mattock.description": "使用模板碎片合成一个远古鹤嘴锄",
+  "advancements.adventure.collect_ancient_mattock.title": "复兴的工具",
+  "advancements.adventure.collect_template_fragment.description": "发现一个模板碎片",
+  "advancements.adventure.collect_template_fragment.title": "一段历史",
+  "biome.newworld.wooded_meadow": "繁茂的草甸",
```

```suggestion
  "biome.newworld.wooded_meadow": "疏林草甸",
```
建议不要使用繁茂，一般是和繁茂洞穴相关的使用
https://zh.minecraft.wiki/w/%E6%81%B6%E5%9C%B0?variant=zh-cn#%E7%96%8F%E6%9E%97%E6%81%B6%E5%9C%B0